freqtrade/strategy/backtest_lookahead_bias_checker.py

```diff
@@ -0,0 +1,241 @@
+# pragma pylint: disable=missing-docstring, W0212, line-too-long, C0103, unused-argument
```

Comment:
well now that's an argument for sure - but isn't the "backtest lookahead bias" better fitted in the "optimize" directory?
also from the command perspective - it's `backtest-` - not `strategy-` ... 

it's just a feeling though - let me know your thoughs as to why you moved it to the strategy dir, maybe?

Reply:
I asked some months ago, frog said that he would group both under strategy_utils so I did
If you want to have the bias checker under optimize then sure, you re the boss I am indifferent about it

Reply:
the pylint line will be gone next time.

Reply:
i'm ... not complaining about pylint lines - we have them at random places - but don't actually use pylint - so it's technically irrelevant to have it or not :laughing: 

---

well i'm ... indecisive - as it'll fit in both areas - but i think things from the strategy package are more likely to be imported (also by mistake, eventually) - so i think keeping that "emptier" makes more sense

Reply:
just say so, will leave it in there until you decided either way :)

Reply:
that pylint sneaked in due to pycharm asking for if it should enter those and I clicked to see what happens - then didnt delete it again